packages/common/package.json

```diff
@@ -21,7 +21,7 @@
     "start": "rollup -c --bundleConfigAsCjs --watch",
     "lint": "eslint . && stylelint \"src/**/*.css\" --allow-empty-input",
     "lint:fix": "eslint . --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
-    "test": "TZ=UTC jest",
+    "test": "TZ=UTC jest && TZ=UTC+02:00 jest src/utils/__tests__/dates.test.ts",
```

Comment:
```suggestion
    "test:UTC": "TZ=UTC jest",
     "test:TZ": "TZ=UTC+02:00 jest src/utils/__tests__/dates.test.ts",
     "test": "npm run test:UTC && npm run test:TZ"
```

Reply:
Moved special time zone tests to a dedicated target as suggest. As for the remaining tests: all the tests across all packages need to run in UTC zone so creating a special target here would be confusing.